Makefile

```diff
@@ -29,10 +29,15 @@ endef
 
 all: bin/ekam
 
-bin/ekam: bin/ekam-bootstrap deps
+bin/ekam: bin/ekam-bootstrap | deps
 	$(call color,building ekam with ekam)
 	@rm -f bin/ekam
 	@CXX="$(CXX)" CXXFLAGS="-std=c++14 $(CXXFLAGS) -pthread" LIBS="-pthread" bin/ekam-bootstrap -j$(PARALLEL)
+	# Ekam rebuilds all the .o files, included the ones used by `ekam-bootstrap`.
+	# This confuses make, which thinks it needs to rebuild `ekam-bootstrap`.
```

Comment:
Probably the bootstrap build should be putting its object files in a different location to avoid confusion.

Of course, any change to the core source code is also going to rebuild the bootstrap -- probably unnecessarily since you already have a fully-functional ekam binary from the previous build to use. Another reason why `make continuous` should require an external `ekam` binary, I think.

Reply:
Can we do this? Touching files to trick make feels icky to me.

Reply:
Oops, I missed this comment. Sure thing, I changed it to use a separate directory for bootstrap artifacts.